internal/processor/title.go

```diff
@@ -82,8 +84,11 @@ func NewTitleSlice() *Titles {
 	return &ts
 }
 
-func (ts *Titles) Add(title string) {
-	title = fmt.Sprintf("*%v*", title)
+func (ts *Titles) Add(title string, cfg *domain.ArrConfig) {
+	if cfg.MatchRelease {
```

Comment:
We can make this a bit simpler to now have the whole `cfg *domain.ArrConfig` and instead just the boolean value of MatchRelease.

```golang
func (ts *Titles) Add(title string, matchRelease bool) {
	if matchRelease {
```

And then for the callers like `t.Add(replace, cfg.MatchRelease)`.

Since we are only interested in the single MatchRelease variable and not anything else of the ArrConfig then it makes little sense to pass the whole struct over the single variable.